packages/hardhat-core/test/internal/cli/autocomplete.ts

```diff
@@ -402,7 +407,7 @@ describe("autocomplete", function () {
     });
   });
 
-  describe("overriden task", () => {
+  describe("override task", () => {
```

Comment:
I think the previous description was correct. This is a suite of tests around an overridden task (well, it had a typo, it was missing a d).

```suggestion
  describe("overridden task", () => {
```